### ISSUE TITLE
Enable submission of an extra study for PMID `28076899`

### ISSUE THREAD

**Issue by eks-ebi** [EBISPOT/goci#635]
**Submission history:**

1. The publication was curated by curators, with 2 entries: (a) multi-ancestry analysis and (b) Asian-only subset
2. The author completed a sumstats-only submission in Nov 2020, and provided sumstats files for entries (a) and (b). Submission ID: `5fabf0c0057c9f0001343cdf`
3. The author would now like to add summary statistics for a 3rd analysis: (c) European-only subset
4. I have replied to the author and asked them to provide metadata about the extra analysis in a template to email back to me

**Task:**

* Determine the best way to allow the author to submit 1 additional study, with summary statistics.
* I (@eks-ebi) can add the new study + metadata manually in curation interface but will wait to hear if that's the best approach

**Challenges:**

* The current deposition interface logic means that because the current studies already have summary statistics, the publication will remain closed for submission even if we add additional studies that do not have summary statistics
* If possible, we do not want to make the author resubmit the sumstats files for (a) and (b), which they already gave us

**Comment by ljwh2** [comment]
Create new study in curation, and then @sajo-ebi  will add manually to appear in prefilled template alongside old studies. Check in with @buniello  for curator perspective. May need to add NR in sumstats fields? Unclear what downstream effects of this will be.

**Comment by buniello** [comment]
Steps should be as follows:
1. re-curation from curation app -- adding new studies
2. publishing new entries and waiting for data release for the submission form to get updated with new entries
3. submission of new sumstats -- published submission. Sajo will need to open the PMID for submission and also take a look at whether the form is rightly populated after new addition (edited) 
Note:
the previously added entries with sumstats should have files showing up in submission form -- though I don't think this really matters at this point in pipeline.

**Comment by ljwh2** [comment]
@eks-ebi please confirm whether this will be ready for developer part within new sprint?

**Comment by eks-ebi** [comment]
yes! I'm working on it now in fact, so should be ready by end of today at latest

**Comment by eks-ebi** [comment]
Steps so far:

1. New study has been created, but I needed to move some top associations from the original study into the newly created one, so am currently waiting for these to map

**Comment by sajo-ebi** [comment]
Hi Elliot, I can add the data from backend , I need the details for study 'GCST012796' in the below format to add the entry in DB -:

`studyAccession
:
"GCST004061"
trait
:
"Sjögren's syndrome"
sampleDescription
:
"585 European ancestry cases, up to 51 African ancestry cases, up to 16..."
hasSummaryStats
:
false`

**Comment by eks-ebi** [comment]
Hi @sajo-ebi, thanks! - here are the details as requested:
`studyAccession : "GCST012796" trait : "Sjögren's syndrome" sampleDescription : "585 European ancestry cases, 1,546 European ancestry controls" hasSummaryStats : false`

**Comment by sajo-ebi** [comment]
@eks-ebi have added the new entry in DB , please create a new sumstats only submission for the Pmid , the prefilled template should have the new study populated

**Comment by eks-ebi** [comment]
@sajo-ebi I just searched for this paper on the Submission page now and it is still closed for submission: https://www.ebi.ac.uk/gwas/deposition/curation_queue (search 28076899) - is it possible that you still have to manually open it for submission?

**Comment by ljwh2** [comment]
Every time deposition sync runs, the submission will be closed - this needs further investigation to make it workable to external users

**Comment by eks-ebi** [comment]
Author has resubmitted, but the submission is stuck on template validation:
https://www.ebi.ac.uk/gwas/deposition/submission/625012423f3b3d0001d45c08

**Comment by eks-ebi** [comment]
@eks-ebi follow up with author - missing p-values in new sumstats files

**Comment by eks-ebi** [comment]
The user has corrected the errors and completed the new submission, and it passed validation yesterday. So this issue *seems* to be resolved.

However, there are some potential new issues:
1. For each of the 2 studies that were submitted by the author the first time around, there are now 2 different sumstats files, which could be misleading for users:
http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST004001-GCST005000/GCST004061/
http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST004001-GCST005000/GCST004062/
2. The 2 files have different file sizes. This could be partly because they have different extensions (.tsv vs .csv), but the difference looks a bit much to be just that. Can we compare them somehow? Or do we need the author to check?
3. The Search UI page for this publication does not display links to the summary stats: https://www.ebi.ac.uk/gwas/publications/28076899. It is also missing from this table: https://www.ebi.ac.uk/gwas/downloads/summary-statistics. Could the double submission be interfering somehow with generating the links?

**Comment by eks-ebi** [comment]
@eks-ebi check with submitter to ask which files we should keep

**Comment by eks-ebi** [comment]
Note: harmonisation pipeline will need to be checked after the correct files are identified

**Comment by eks-ebi** [comment]
Discussed with author (see RT: https://helpdesk.ebi.ac.uk/Ticket/Display.html?id=566111&results=1e5f17da461efa716915184955a0136c) and confirmed that older .tsv files should be discarded and the more recent .csv files should be kept.

I have now manually deleted the old files from the ftp.

**Comment by eks-ebi** [comment]
@jdhayhurst could you please look at retriggering the harmonisation pipeline for this publication? Now that the extra sumstats files have been removed, it should be able to run as normal

**Comment by jdhayhurst** [comment]
added to queue
